include/simdjson/simdjson_version.h

```diff
@@ -1,26 +1,13 @@
 // /include/simdjson/simdjson_version.h automatically generated by release.py,
 // do not change by hand
-#ifndef SIMDJSON_SIMDJSON_VERSION_H
-#define SIMDJSON_SIMDJSON_VERSION_H
-
-/** The version of simdjson being used (major.minor.revision) */
-#define SIMDJSON_VERSION 0.2.1
-
+#ifndef SIMDJSON_INCLUDE_SIMDJSON_VERSION
+#define SIMDJSON_INCLUDE_SIMDJSON_VERSION
+#define SIMDJSON_VERSION 0.3.0
 namespace simdjson {
 enum {
-  /**
-   * The major version (MAJOR.minor.revision) of simdjson being used.
-   */
   SIMDJSON_VERSION_MAJOR = 0,
-  /**
```

Comment:
Grr. I guess it's automatically generated and can't be helped.

Reply:
It can be helped. I will fix this.

Reply:
There you go. Things are now identical to the hand-tuned version you crafted. Now, of course, you see the warning about the file being automatically generated? You are not going to tweak it again, are you? :-)

(Releases need to be scripted otherwise it is too easy for errors to slip in and they are rather tragic. You don't want the version number to be a subject of confusion.)